coldfront/core/utils/validate.py

```diff
@@ -70,13 +60,9 @@ def __init__(self, value):
 
     def validate_billing_cycle(self):
         try:
-            validate = validators.OneOf(BILLING_CYCLE_OPTIONS)
+            validate = validators.OneOf([option[0] for option in BILLING_CYCLE_OPTIONS])

```

Comment:
[Remark] The list-comprehension was nicely done.

Reply:
Thanks